seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-2/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcTrinoIT.java

```diff
@@ -0,0 +1,118 @@
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.TestTemplate;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.lifecycle.Startables;
+import org.testcontainers.utility.DockerLoggerFactory;
+
+import com.google.common.collect.Lists;
+import lombok.extern.slf4j.Slf4j;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Stream;
+
+import static org.awaitility.Awaitility.given;
+
+@Slf4j
+public class JdbcTrinoIT extends TestSuiteBase implements TestResource {
+
+    private static final String DOCKER_IMAGE = "trinodb/trino:445";
+    private static final String DRIVER_CLASS = "io.trino.jdbc.TrinoDriver";
+    private static final String HOST = "e2e_tino";
+    private static final String URL = "jdbc:trino://%s:5236";
+
+    private Connection jdbcConnection;
+    private GenericContainer<?> dbServer;
+
+    @BeforeAll
+    @Override
+    public void startUp() throws Exception {
+        dbServer =
+                new GenericContainer<>(DOCKER_IMAGE)
+                        .withNetwork(NETWORK)
+                        .withNetworkAliases(HOST)
+                        .withLogConsumer(
+                                new Slf4jLogConsumer(DockerLoggerFactory.getLogger(DOCKER_IMAGE)));
+        dbServer.setPortBindings(Lists.newArrayList(String.format("%s:%s", 5236, 8080)));
+        Startables.deepStart(Stream.of(dbServer)).join();
+        log.info("Trino container started");
+        // wait for trino fully start
+        Class.forName(DRIVER_CLASS);
+        given().ignoreExceptions()
+                .await()
+                .atMost(60, TimeUnit.SECONDS)
+                .untilAsserted(this::initializeJdbcConnection);
+        initializeJdbcTable();
+    }
+
+    private void initializeJdbcConnection() throws SQLException {
+        Properties properties = new Properties();
+        properties.setProperty("user", "trino");
+        jdbcConnection =
+                DriverManager.getConnection(String.format(URL, dbServer.getHost()), properties);
+    }
+
+    private void initializeJdbcTable() {
+        // no things
+    }
+
+    @AfterAll
+    @Override
+    public void tearDown() throws Exception {
+        if (jdbcConnection != null) {
+            jdbcConnection.close();
+        }
+        if (dbServer != null) {
+            dbServer.close();
+        }
+    }
+
+    @TestTemplate
+    @DisplayName("JDBC-Trino end to end test")
+    public void testJdbcTrino(TestContainer container) {
+        assertHasData();
+        log.info(container.toString());
+    }
```

Comment:
I can not find the test case of trino. Please add a test case for it. You can refer https://github.com/apache/seatunnel/blob/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-6/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcHanaIT.java#L105